#!/usr/bin/env python
PACKAGE = "hausuebung2_semjon_eschweiler"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("k_linear",    double_t,    0, "Control parameter linear", 0.125,  -50, 50)
gen.add("k_alpha", double_t, 0, "Control parameter angular", 0.4,  -50, 50)
gen.add("k_beta", double_t, 0, "Control parameter angular", 0.14,  -50, 50)
gen.add("x_goal",    double_t,    0, "x goal in m", 3,  -10, 10)
gen.add("y_goal",    double_t,    0, "y goal in m", 0,  -10, 10)
gen.add("th_goal",    double_t,    0, "Theta goal in degrees", 0, -180, 180)
gen.add("start_on_check",   bool_t,   0, "Starts navigation to goal pose, when check is clicked",  False)
gen.add("reset_on_check",   bool_t,   0, "Resets the simulation",  False)

exit(gen.generate(PACKAGE, "hausuebung2_semjon_eschweiler", "dynamicParams"))